iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -293,10 +293,10 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
 
                 let message = ChatClient.shared.message(cid: cid, id: messageID)
 
-                if indexPath.section == 0 {
+                if !message.isFromCurrentUser && state.detailShown {
                     self.frontmostNonUserMessage = message
                 }
```

Comment:
Looks like you removed the guard that checks to see if the details are shown. We still need to check that at some point though to determine if the message is frontmost.